src/lib/protobuf-c/protobuf-c.h

```diff
@@ -200,6 +200,7 @@ size_t foo__bar__baz_bah__pack_to_buffer
 #include <limits.h>
 #include <stddef.h>
 #include <stdint.h>
+#include "osal_common.h"
```

Comment:
Need to update this appropriately after concluding osal file/dir structure mentioned in the previous comment.